org.xfce.ristretto.yml

```diff
@@ -11,6 +11,10 @@ finish-args:
   # the open dialog is native, but we still need this to open files via the command line
   # and drag and drop, and to delete files
   - --filesystem=host
+  # FIXME: necessary for Tumbler to emit signals, see
+  # https://github.com/flathub/flathub/issues/2770
+  # https://github.com/flathub/org.xfce.ristretto/pull/7
+  - --socket=session-bus
```

Comment:
This is bad, you should only give it access to specific dbus interfaces the application should be capable of talking to, using --talk-name

Reply:
Ah I see, you need to use --own-name in your case

Reply:
No, it doesn't work either. Please see flathub/flathub#2770.